distribution/changelog.txt

```diff
@@ -3,7 +3,7 @@
 - Fix: [#5579] Network desync immediately after connecting.
 - Fix: [#7884] Unfinished preserved rides can be demolished with quick demolish.
 - Fix: [#8873] Potential crash when placing footpaths.
-- Change: [#8688] Move common actions from debug menu into cheats menu.
+- Fix: [#8900] Synchronize peep tracking to prevent desyncs.
```

Comment:
You unintentionally removed another entry.

Reply:
Hm thats strange, I recall keeping it when the rebase had conflicts, ill fix it.

Reply:
... now you moved it up the list :/
Can you just keep it where it was, i.e. after the fixes?

Reply:
It was ordered like that from the previous release, but okay.

Reply:
You're right, that was a mistake on my part.